rskj-core/src/main/java/co/rsk/net/sync/PeersInformation.java

```diff
@@ -51,11 +51,13 @@ public class PeersInformation {
     private final PeerScoringManager peerScoringManager;
     private final Comparator<Map.Entry<Peer, SyncPeerStatus>> peerComparator;
     private Map<Peer, SyncPeerStatus> peerStatuses = new HashMap<>();
+    private final double percentageOfPeersToConsiderInRandomSelection;
```

Comment:
it seems that this config could be part of `SyncConfiguration` class? Also can it be more shorter name? :) Eg. `topBest`?

Reply:
Done